pkgs/applications/misc/chrysalis/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, appimageTools, fetchurl }:
+
+let
+  pname = "chrysalis";
+  version = "0.7.9";
+in appimageTools.wrapType2 rec {
+  name = "${pname}-${version}-binary";
+
+  src = fetchurl {
+    url = "https://github.com/keyboardio/${pname}/releases/download/${pname}-${version}/${pname}-${version}.AppImage";
+    sha256 = "12w4vv7dwfpvxpc8kpfas90y7yy8mb8dj2096z3vw1bli5lrn3zi";
+  };
+
+  multiPkgs = null;
+  extraPkgs = p: (appimageTools.defaultFhsEnvArgs.multiPkgs p) ++ [
```

Comment:
Why do you transfer the packages from `multiPkgs` to `extraPkgs`?

Reply:
Setting `multiPkgs = null` for `appimageTools.wrapType2` disables multilib packages (this is a pure 64-bit package).  `appimageTools.defaultFhsEnvArgs.multiPkgs` holds the list of packages required for an AppImage, which would normally be pulled in by `multiPkgs`, though, and now need to be explicitly pulled in as non-multilib packages.

This is a quite common construct in `nixpkgs` for AppImages (I admit to not having thought this up myself ;)